admin/widgets-options-filters-admin.php

```diff
@@ -0,0 +1,51 @@
+<?php
+/**
+ * @package Polylang
+ */
+
+/**
+ * Class PLL_Widgets_Filters
+ *
+ * Adds new options to {@see https://developer.wordpress.org/reference/classes/wp_widget/ WP_Widget} and saves them.
+ */
+class PLL_Widgets_Options_Filters_Admin extends PLL_Widgets_Options_Filters {
```

Comment:
Shouldn't we name it PLL_Admin_Widgets_Options_Filters ? To respect how we name all the other class